CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+cmake_minimum_required(VERSION 3.14)
+project(tangerine)
+
+if (MSVC)
+	message(FATAL_ERROR 
```

Comment:
We probably need to change something in the vcxproj so that Visual Studio doesn't automatically try to load this if CMake is installed. I have not tested that yet.

Reply:
Alternately, I could try to get CMake working on Windows too. It ought to be pretty simple. But this way, the build system you're used to doesn't change.